### PR TITLE
fix: remove unnecessary NetworkVariable in 2DSpaceShooter

### PR DESCRIPTION
**Pull request by fernando-cortez**
### Description
Addressing the issue [here](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/issues/128).
This PR removes an unnecessary NetworkVariable from 2DSpaceShooter.
Benefit: reducing network bandwidth used to sync variable to clients.

### Issue Number(s)
Original [issue](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/issues/128).

### Contribution checklist
 - [ N/A ] Tests have been added for the project and/or any internal package
 - [x] Release notes have been added to the [project changelog](../CHANGELOG.md) file
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ N/A ] JIRA ticket ID is in the PR title or at least one commit message
 - [ N/A ] Include the ticket ID number within the body message of the PR to create a hyperlink

### CODE REVIEW

**Comment by jilfranco-unity** [comment]
added a quick comment, but otherwise this all makes sense!

**Review by jilfranco-unity — CHANGES_REQUESTED** [review]

File: Basic/2DSpaceShooter/Assets/Scripts/ShipControl.cs [hunk 1]
```diff
@@ -107,7 +107,7 @@ public class ShipControl : NetworkBehaviour
     float m_OldSpin = 0;
 
     // server movement
-    private NetworkVariable<float> m_Thrusting = new NetworkVariable<float>();
+    float m_Thrusting;
```
Comment (jilfranco-unity) [inline Basic/2DSpaceShooter/Assets/Scripts/ShipControl.cs:110]: While we're here, might be better to rename to `m_Thrust` or `m_Speed`

Comment (fernando-cortez) [reply Basic/2DSpaceShooter/Assets/Scripts/ShipControl.cs]: Done!
I had to rename a particle system that used the `m_Thrust` name, but this is clearer yeah.

**Review by jilfranco-unity — APPROVED** [review]
looks good, validated that particle system still works as intended

**Pull request merged**